Cargo.toml

```diff
@@ -27,8 +27,8 @@ crossbeam-utils = "^0.7.0"
 prometheus = { version = "0.9", features = ["push"], optional = true }
 hyper = { version = "0.13", optional = true }
 tokio = { version = "0.2", features = ["macros"], optional = true }
-wapc = "0.8.1"
-wascc-codec = "0.7.2"
+wapc = "0.9"
+wascc-codec = "0.8"
```

Comment:
Note: This won't build until wascc-codec has gotten been released

Reply:
There's a transitive dependency problem on wascc-codec that won't be fixed until the lattice-client dependency gets bumped to 0.0.3